src/prfmodel/stimulus.py

```diff
@@ -249,3 +257,64 @@ def create_2d_bar_stimulus(  # noqa: PLR0913 (too many arguments)
             grid=grid,
             dimension_labels=dimension_labels,
         )
+
+
+def make_video(stimulus: Stimulus, title: str | None = None, kwargs: dict | None = None) -> str:
```

Comment:
Here, I would probably use `**kwargs` without a default and say in the docstring that all keyword args are passed to `ArtistAnimation`. Then users can use keyword args directly and don't need to wrap them in a dict. Then, I would also make the default keyword args explicit (e.g., `interval`, ...).